tests/unit/s2n_client_signature_algorithms_extension_test.c

```diff
@@ -125,21 +117,14 @@ int main(int argc, char **argv)
                 EXPECT_SUCCESS(s2n_stuffer_write_uint16(&signature_algorithms_extension, sig_hash_algs.iana_list[i]));
         }
 
-        struct s2n_array *parsed_extensions = s2n_array_new(sizeof(struct s2n_client_hello_parsed_extension));
-        struct s2n_client_hello_parsed_extension *parsed_named_group_extension = NULL;
-        EXPECT_OK(s2n_array_pushback(parsed_extensions, (void **)&parsed_named_group_extension));
-        parsed_named_group_extension->extension_type = TLS_EXTENSION_SIGNATURE_ALGORITHMS;
-        parsed_named_group_extension->extension = signature_algorithms_extension.blob;
-
         /* If a valid algorithm is offered among unknown algorithms, the valid one should be chosen */
-        EXPECT_SUCCESS(s2n_client_extensions_recv(conn, parsed_extensions));
+        EXPECT_SUCCESS(s2n_client_signature_algorithms_extension.recv(conn, &signature_algorithms_extension));
```

Comment:
nit: not related to your changes but could be more specific in the comment on what algorithm, same as above.

Reply:
https://github.com/awslabs/s2n/pull/1956#discussion_r432202305